decidim-admin/app/assets/javascripts/decidim/admin/newsletters.js.es6

```diff
@@ -66,5 +66,14 @@ $(() => {
         }
       });
     })
+
+    $(".form.newsletter_deliver").on("change", function() {
+      let $data = $form.serializeJSON().newsletter;
+      let $id = $form.attr("id").match(/\d+/)[0];
+      let $url = `/admin/newsletters/${$id}/recipients_count`;
```

Comment:
I'd add the path as a `data` attribute of the form in `decidim-admin/app/views/decidim/admin/newsletters/select_recipients_to_deliver.html.erb`:

```erb
data: {"recipients-count-newsletter-path": recipients_count_newsletter_path(@newsletter)}
```

 and here get that value:

```js
let $url = $form.data("recipients-count-newsletter-path");
```